.gitignore

```diff
@@ -11,6 +11,10 @@ node_modules
 target
 .bsp
 
+# npm/yarn install files
```

Comment:
We want to lock the yarn deps, so you can remove that ignore below.  Should update this comment to indicate that we only use yarn for package management, so the npm lock is ignored here.

The npm lock shouldn't be generated given our dev docs, but just in case someone does we will ignore it.